packages/EasyAsync/src/Implementations/Doctrine/DBAL/JobLogPersister.php

```diff
@@ -34,7 +34,7 @@ final class JobLogPersister extends AbstractPersister implements JobLogPersister
     public function __construct(
         Connection $conn,
         DateTimeGeneratorInterface $dateTime,
-        UuidGeneratorInterface $uuid,
+        UuidV4GeneratorInterface $uuid,
```

Comment:
Same here, inject `RandomGeneratorInterface` and call `uuidV4()`.

Reply:
Done.